sympy/logic/boolalg.py

```diff
@@ -736,6 +736,10 @@ def eval(cls, arg):
         if isinstance(arg, GreaterThan):
             return StrictLessThan(*arg.args)
 
+    @property
+    def canonical(self):
+        return self
```

Comment:
I would suggest `return self.func(*Eq(*self.args).canonical.args)` so whatever is done to make Eq canonical will be done for Ne.